Framework/Algorithms/src/MonteCarloAbsorption.cpp

```diff
@@ -108,6 +448,26 @@ void MonteCarloAbsorption::init() {
 
   InterpolationOption interpolateOpt;
   declareProperty(interpolateOpt.property(), interpolateOpt.propertyDoc());
+  declareProperty("SparseInstrument", false, "Enable simulation on special "
+                                             "instrument with a sparse grid of "
+                                             "detectors interpolating the "
+                                             "results to the real instrument.");
+  auto twoOrMore = boost::make_shared<Kernel::BoundedValidator<int>>();
+  twoOrMore->setLower(2);
```

Comment:
As we have tested once, I think it is better to set an odd default value (so 3) for the latitude points. Supposedly most of the instruments will be more or less symmetric around the equatorial, where with 2 points only one would simulate the upper and lower bands, and get a systematic shift after interpolating to the equator.